Index.html

```diff
@@ -0,0 +1,15 @@
+<html>
+    <head>
+        <title>Hello Microverse exercise</title>
+    </head>
+    <body>
+        <div class="heading">
+                <h1>Hello Microverse</h1>
```

Comment:
- [ ] The style applied to this heading class will affect every element inside this `<div>,` Please add this class to `<h1>` instead of `<div>` so the styles will only be applied to the `<h1>` heading.